set up SIG-etcd (#7372)
kubernetes/community: sig-etcd/README.md

<!---
This is an autogenerated file!

Please do not edit this file directly, but instead make changes to the
sigs.yaml file in the project root.

To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
--->
# etcd Special Interest Group

> Make etcd the best production-ready store for building cloud-native distributed systems and managing cloud-native infrastructure via orchestrators like Kubernetes.

Reviewer: ```suggestion
etcd is a production-ready store for building cloud-native distributed systems and managing cloud-native infrastructure via orchestrators like Kubernetes.
```

I'm not explicitly saying etcd isn't the best, but I don't think best is even a goal of Kubernetes itself. etcd is just one way to implement this type of key value store, but it's one that's currently critical path to the upstream implementation of Kubernetes and that's why we want to bring it under the larger Kubernetes project.

Reviewer 2: Done.

Etcd should provide distributed system primitives** (such as distributed locking and leader election) that allow users to **create scalable, highly available and fault-tolerant systems.
Etcd is the place to store the infrastructure configuration, not only as part of Kubernetes, but also as a standalone solution.

The [charter](charter.md) defines the scope and governance of the etcd Special Interest Group.

## Meetings
*Joining the [mailing list](https://groups.google.com/g/etcd-dev) for the group will typically add invites for the following meetings to your calendar.*
* Regular SIG Meeting: [Thursdays at 9:30 PT (Pacific Time)](https://meet.google.com/umg-nrxn-qvs) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:30&tz=PT%20%28Pacific%20Time%29).
  * [Meeting notes and Agenda](https://docs.google.com/document/d/16XEGyPBisZvmmoIHSZzv__LoyOeluC5a4x353CX0SIM/edit?usp=sharing).

## Leadership

> ### Chairs

Author: ```suggestion
### Interim Chairs (official chairs to be elected once SIG is in place)
```
